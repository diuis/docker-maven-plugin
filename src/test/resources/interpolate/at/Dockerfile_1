FROM @base@
LABEL name ${name} @age@ ${ext} blub @unknown@ @
ENV @name@ @ext@
