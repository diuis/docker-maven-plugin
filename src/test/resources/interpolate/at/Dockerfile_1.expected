FROM java
LABEL name ${name} 42 ${ext} blub @unknown@ @
ENV guenther png
